ecommerce-ui/src/AirbnbButton.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import Button from "@material-ui/core/Button";
+import PropTypes from "prop-types";
+
+export default class AirbnbButton extends React.Component {
+  render() {
+    const { onClick, idx, buttonText } = this.props;
+
+    return (
+        <Button
+          size="small"
+          onClick={() => onClick(idx)}
```

Comment:
A good example of having to use an anonymous function pointing to the actual on click function so we can pass a variable to the callback function.